src/core-interfaces.ts

```diff
@@ -295,6 +295,18 @@ export interface TextBaseProps {
 		 * @example 10 // Indents text 10 points from bullet
 		 */
 		indent?: number
+		/**
+		 * ratio between `before text` indent and hanging indent
+		 *
+		 * passing this option will opt into different bullet behaviour:
+		 * `indent` will be treated as the `before text` indent
+		 * and the hanging indent will be calculated from this value.
+		 *
+		 * (normally, `indent` is the hanging indent and the `before text` indent is 2, 3, 4, 5, etc. times that value)
+		 * @example 5/7 or 0.71428
+		 * // hanging indent value will be 5/7 the size of the base `before text` indent value
+		 */
+		hangingIndentRatio?: number
```

Comment:
<!-- Ask the author for clarification or investigation.  -->
**question (non-blocking):** just a clarifying question for personal understandin
- ratio of 1 is the existing behaviour, 
- ratio < 1 then the bullet gets closer to the text
- ratio > 1 then the bullet gets further away from text

Reply:
yeah, pretty much. when further away the bullet will move left to max(left pos, boundary of text box) without pushing the start of the bullet text content over to make the gap as wide as specified (since that's controlled by `indent` / `before text margin` / `marL`)

when closer, it will get as close as possible but with large bullets like `xvii.` it may get closer than specified, up to directly beside, since PPT maintains the distance between the start of the bullet and the start of the text, not the `.` and the start of the text.

Technically, ratio of 1 is the default behaviour when typing _in PPT_.
I _think_ 0.5 would be the existing behaviour of _PptxGenJS_ but I haven't tested it